CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.5.0 / 2018-05-20
+
+Breaking change:
+* Flags now require double-dash.
+* The persistence storage format has been updated.  Upgrade is transparent, but downgrade to 0.4.0 and prior is unsupported.
+* Persistence storage prior to 0.1.0 is unsupported.
+
+* [CHANGE] Debug level logging now shows client-induced errors #123
```

Comment:
This is an ENHANCEMENT if at all. Log lines are not part of the interface, especially not debug log lines, so it's not a change to the contract.

Reply:
Fixed.